
8/3/2019 5:52:28 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 10:46:24 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 10:49:38 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 10:49:40 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 10:52:33 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 10:52:47 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 10:57:22 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:03:09 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/3/2019 11:07:11 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/3/2019 11:10:50 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:12:36 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:13:44 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:14:50 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:16:40 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/3/2019 11:17:54 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:19:17 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:20:42 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/3/2019 11:21:23 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:22:17 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:23:40 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/3/2019 11:25:17 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:27:33 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:30:56 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:35:05 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/3/2019 11:35:40 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:35:50 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:35:57 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:36:58 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:38:17 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/3/2019 11:38:35 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:39:24 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:39:51 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:40:32 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:40:41 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:41:04 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:42:11 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:43:16 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:44:43 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:45:09 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:45:45 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:46:06 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:46:18 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:46:49 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:49:57 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:50:35 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:52:50 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/3/2019 11:55:52 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:56:07 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/3/2019 11:58:15 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/4/2019 12:00:06 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:01:25 AM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/4/2019 12:05:03 AM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/4/2019 12:06:07 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:09:02 AM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/4/2019 12:18:58 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:19:48 AM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/4/2019 12:20:54 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:22:10 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:23:09 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:24:48 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:26:10 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:26:38 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:26:55 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:28:03 AM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 225


8/4/2019 12:29:21 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:29:29 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:29:40 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:34:21 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:38:47 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/4/2019 12:38:48 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:36:30 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:39:26 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:39:45 AM
System.IO.IOException: Unable to read data from the transport connection: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/6/2019 9:41:31 AM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 178


8/6/2019 9:43:31 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:43:53 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:44:21 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:46:42 AM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 225


8/6/2019 9:47:00 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:47:23 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:47:29 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:49:11 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:49:57 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:50:31 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:53:32 AM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 225


8/6/2019 9:54:52 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 9:55:04 AM
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at DemoSocket.Client.SendAllRoom(Int32 id_Room, String nhan) in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 169
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 206


8/6/2019 10:00:31 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/6/2019 10:00:32 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179


8/8/2019 9:53:38 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/8/2019 9:53:44 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/8/2019 10:02:52 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/8/2019 10:02:53 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/8/2019 10:12:42 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/8/2019 10:12:43 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/8/2019 10:14:07 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/8/2019 10:14:24 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 10:14:59 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 10:15:02 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 10:21:18 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 10:21:18 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 10:39:38 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 10:39:46 AM
System.IO.IOException: Unable to read data from the transport connection: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 186


8/9/2019 10:45:54 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 10:45:55 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 10:47:27 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 10:47:27 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 10:53:16 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 10:55:57 AM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 238


8/9/2019 11:01:04 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 11:05:19 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 11:05:47 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 3:47:32 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 3:58:20 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 3:58:30 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 3:59:48 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 4:11:20 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 4:14:11 PM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 238


8/9/2019 4:14:18 PM
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at DemoSocket.Client.SendAllRoom(Int32 id_Room, String nhan) in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 177
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 261


8/9/2019 4:15:50 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 4:15:51 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 4:15:52 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 4:24:08 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 4:27:49 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 4:29:24 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 4:39:10 PM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 238


8/9/2019 4:49:34 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 4:49:47 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 4:51:04 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 4:52:08 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:02:40 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:02:46 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:16:57 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:17:47 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:18:57 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:20:25 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:25:30 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:25:31 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:25:33 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:25:42 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:51:36 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:51:37 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:51:38 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:51:42 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 5:59:43 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 186


8/9/2019 6:00:27 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:11:47 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:12:12 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:23:56 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:25:18 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:25:54 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:28:50 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:35:24 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:39:16 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:41:18 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:42:31 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:47:48 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:47:52 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:47:55 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 6:47:56 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 10:50:09 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 11:01:58 PM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 238


8/9/2019 11:05:37 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 11:05:41 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 11:06:53 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 186


8/9/2019 11:08:40 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 11:09:19 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 11:10:57 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 11:18:01 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 11:18:50 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/9/2019 11:20:53 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 12:46:19 AM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 238


8/10/2019 12:53:23 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 12:53:24 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 12:59:59 AM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 186


8/10/2019 12:59:59 AM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 186


8/10/2019 1:06:21 AM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 186


8/10/2019 1:07:32 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 1:09:24 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 1:09:25 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 9:43:11 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 9:44:21 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 9:48:51 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 9:48:57 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 9:56:07 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 9:58:09 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 9:58:21 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 10:01:55 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 10:02:21 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 10:02:25 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 10:13:52 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 10:14:49 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 10:59:02 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 11:06:44 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 6:38:25 PM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 238


8/10/2019 6:55:28 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 11:50:41 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/10/2019 11:56:00 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 186


8/11/2019 12:09:29 AM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 238


8/11/2019 12:49:22 AM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 186


8/11/2019 6:15:29 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:16:20 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:17:06 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:17:51 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:23:58 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:25:07 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:28:45 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:33:04 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:42:18 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:43:09 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:43:21 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:43:43 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:43:56 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:45:25 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:45:58 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:46:16 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:47:33 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:57:16 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 6:58:03 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/11/2019 7:07:13 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 187


8/13/2019 12:00:32 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:02:20 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:03:51 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:04:58 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:05:48 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:08:42 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:09:19 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:10:12 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:11:23 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:12:13 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:13:09 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:14:27 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:16:05 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:19:37 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:29:06 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:30:52 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:31:59 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:41:12 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 12:59:14 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:02:08 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:02:48 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:06:14 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:08:07 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:11:58 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:15:42 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:20:58 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:22:20 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:23:54 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:25:51 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:25:52 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:29:45 AM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 272


8/13/2019 1:29:58 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:30:33 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:30:38 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:32:02 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 1:32:05 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 9:16:40 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 9:17:48 AM
System.Data.Entity.Core.EntityCommandExecutionException: An error occurred while executing the command definition. See the inner exception for details. ---> System.Data.SqlClient.SqlException: Cannot roll back addmoney. No transaction or savepoint of that name was found.
Transaction count after EXECUTE indicates a mismatching number of BEGIN and COMMIT statements. Previous count = 1, current count = 2.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.<Reader>b__c(DbCommand t, DbCommandInterceptionContext`1 c)
   at System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext,TResult](TTarget target, Func`3 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.Reader(DbCommand command, DbCommandInterceptionContext interceptionContext)
   at System.Data.Entity.Internal.InterceptableDbCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   --- End of inner exception stack trace ---
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.Execute(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteNonQuery()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunctionCommand(EntityCommand entityCommand)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__4a()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__49()
   at System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunction(String functionName, ObjectParameter[] parameters)
   at DemoSocket.Models.DatabaseGameBaiEntities.congTien(Nullable`1 amount, Nullable`1 id) in D:\C#_project\demo\DemoSocket\DemoSocket\Models\PlayerModel.Context.cs:line 60
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 292


8/13/2019 9:19:06 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 9:25:45 AM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 272


8/13/2019 9:27:50 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 9:28:44 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 9:29:32 AM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 272


8/13/2019 9:29:44 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 9:31:12 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 9:40:07 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 9:40:15 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 9:40:24 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 2:13:10 PM
System.Data.Entity.Core.EntityCommandExecutionException: An error occurred while executing the command definition. See the inner exception for details. ---> System.Data.SqlClient.SqlException: Cannot roll back addmoney. No transaction or savepoint of that name was found.
Transaction count after EXECUTE indicates a mismatching number of BEGIN and COMMIT statements. Previous count = 1, current count = 2.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.<Reader>b__c(DbCommand t, DbCommandInterceptionContext`1 c)
   at System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext,TResult](TTarget target, Func`3 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.Reader(DbCommand command, DbCommandInterceptionContext interceptionContext)
   at System.Data.Entity.Internal.InterceptableDbCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   --- End of inner exception stack trace ---
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.Execute(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteNonQuery()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunctionCommand(EntityCommand entityCommand)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__4a()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__49()
   at System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunction(String functionName, ObjectParameter[] parameters)
   at DemoSocket.Models.DatabaseGameBaiEntities.truTien(Nullable`1 amount, Nullable`1 id) in D:\C#_project\demo\DemoSocket\DemoSocket\Models\PlayerModel.Context.cs:line 99
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 306


8/13/2019 2:17:20 PM
System.IO.IOException: Unable to write data to the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Send(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamWriter.Flush(Boolean flushStream, Boolean flushEncoder)
   at System.IO.StreamWriter.Write(Char[] buffer, Int32 index, Int32 count)
   at System.IO.TextWriter.WriteLine(String value)
   at DemoSocket.Client.Send(String input) in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 152
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 278


8/13/2019 2:18:22 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 2:18:28 PM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 272


8/13/2019 2:18:51 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 2:19:11 PM
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at DemoSocket.Client.SendAllRoom(Int32 id_Room, String nhan) in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 179
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 236


8/13/2019 2:19:26 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 2:21:06 PM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 272


8/13/2019 2:21:28 PM
System.IO.IOException: Unable to write data to the transport connection: An established connection was aborted by the software in your host machine. ---> System.Net.Sockets.SocketException: An established connection was aborted by the software in your host machine
   at System.Net.Sockets.Socket.Send(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamWriter.Flush(Boolean flushStream, Boolean flushEncoder)
   at System.IO.StreamWriter.Write(Char[] buffer, Int32 index, Int32 count)
   at System.IO.TextWriter.WriteLine(String value)
   at DemoSocket.Client.Send(String input) in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 152
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 278


8/13/2019 2:22:18 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 2:22:18 PM
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 200


8/13/2019 2:22:22 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 2:24:02 PM
System.Data.Entity.Core.EntityCommandExecutionException: An error occurred while executing the command definition. See the inner exception for details. ---> System.Data.SqlClient.SqlException: Cannot roll back addmoney. No transaction or savepoint of that name was found.
Transaction count after EXECUTE indicates a mismatching number of BEGIN and COMMIT statements. Previous count = 1, current count = 2.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.<Reader>b__c(DbCommand t, DbCommandInterceptionContext`1 c)
   at System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext,TResult](TTarget target, Func`3 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.Reader(DbCommand command, DbCommandInterceptionContext interceptionContext)
   at System.Data.Entity.Internal.InterceptableDbCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   --- End of inner exception stack trace ---
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.Execute(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteNonQuery()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunctionCommand(EntityCommand entityCommand)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__4a()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__49()
   at System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunction(String functionName, ObjectParameter[] parameters)
   at DemoSocket.Models.DatabaseGameBaiEntities.truTien(Nullable`1 amount, Nullable`1 id) in D:\C#_project\demo\DemoSocket\DemoSocket\Models\PlayerModel.Context.cs:line 99
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 306


8/13/2019 2:33:59 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 2:34:24 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 201


8/13/2019 2:34:59 PM
System.Data.Entity.Core.EntityCommandExecutionException: An error occurred while executing the command definition. See the inner exception for details. ---> System.Data.SqlClient.SqlException: Cannot roll back addmoney. No transaction or savepoint of that name was found.
Transaction count after EXECUTE indicates a mismatching number of BEGIN and COMMIT statements. Previous count = 1, current count = 2.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.<Reader>b__c(DbCommand t, DbCommandInterceptionContext`1 c)
   at System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext,TResult](TTarget target, Func`3 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.Reader(DbCommand command, DbCommandInterceptionContext interceptionContext)
   at System.Data.Entity.Internal.InterceptableDbCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   --- End of inner exception stack trace ---
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.Execute(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteNonQuery()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunctionCommand(EntityCommand entityCommand)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__4a()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__49()
   at System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunction(String functionName, ObjectParameter[] parameters)
   at DemoSocket.Models.DatabaseGameBaiEntities.truTien(Nullable`1 amount, Nullable`1 id) in D:\C#_project\demo\DemoSocket\DemoSocket\Models\PlayerModel.Context.cs:line 99
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 306


8/13/2019 2:42:45 PM
System.Data.Entity.Core.EntityCommandExecutionException: An error occurred while executing the command definition. See the inner exception for details. ---> System.Data.SqlClient.SqlException: Cannot roll back addmoney. No transaction or savepoint of that name was found.
Transaction count after EXECUTE indicates a mismatching number of BEGIN and COMMIT statements. Previous count = 1, current count = 2.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.<Reader>b__c(DbCommand t, DbCommandInterceptionContext`1 c)
   at System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext,TResult](TTarget target, Func`3 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.Reader(DbCommand command, DbCommandInterceptionContext interceptionContext)
   at System.Data.Entity.Internal.InterceptableDbCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   --- End of inner exception stack trace ---
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.Execute(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteNonQuery()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunctionCommand(EntityCommand entityCommand)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__4a()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__49()
   at System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunction(String functionName, ObjectParameter[] parameters)
   at DemoSocket.Models.DatabaseGameBaiEntities.truTien(Nullable`1 amount, Nullable`1 id) in D:\C#_project\demo\DemoSocket\DemoSocket\Models\PlayerModel.Context.cs:line 99
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 306


8/13/2019 2:44:40 PM
System.Data.Entity.Core.EntityCommandExecutionException: An error occurred while executing the command definition. See the inner exception for details. ---> System.Data.SqlClient.SqlException: Cannot roll back addmoney. No transaction or savepoint of that name was found.
Transaction count after EXECUTE indicates a mismatching number of BEGIN and COMMIT statements. Previous count = 1, current count = 2.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.<Reader>b__c(DbCommand t, DbCommandInterceptionContext`1 c)
   at System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext,TResult](TTarget target, Func`3 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.Reader(DbCommand command, DbCommandInterceptionContext interceptionContext)
   at System.Data.Entity.Internal.InterceptableDbCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   --- End of inner exception stack trace ---
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.Execute(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteNonQuery()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunctionCommand(EntityCommand entityCommand)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__4a()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__49()
   at System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunction(String functionName, ObjectParameter[] parameters)
   at DemoSocket.Models.DatabaseGameBaiEntities.truTien(Nullable`1 amount, Nullable`1 id) in D:\C#_project\demo\DemoSocket\DemoSocket\Models\PlayerModel.Context.cs:line 99
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 306


8/13/2019 2:47:08 PM
System.Data.Entity.Core.EntityCommandExecutionException: An error occurred while executing the command definition. See the inner exception for details. ---> System.Data.SqlClient.SqlException: Cannot roll back addmoney. No transaction or savepoint of that name was found.
Transaction count after EXECUTE indicates a mismatching number of BEGIN and COMMIT statements. Previous count = 1, current count = 2.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.<Reader>b__c(DbCommand t, DbCommandInterceptionContext`1 c)
   at System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext,TResult](TTarget target, Func`3 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.Reader(DbCommand command, DbCommandInterceptionContext interceptionContext)
   at System.Data.Entity.Internal.InterceptableDbCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   --- End of inner exception stack trace ---
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.Execute(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteNonQuery()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunctionCommand(EntityCommand entityCommand)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__4a()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__49()
   at System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunction(String functionName, ObjectParameter[] parameters)
   at DemoSocket.Models.DatabaseGameBaiEntities.truTien(Nullable`1 amount, Nullable`1 id) in D:\C#_project\demo\DemoSocket\DemoSocket\Models\PlayerModel.Context.cs:line 99
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 306


8/13/2019 2:48:12 PM
System.Data.Entity.Core.EntityCommandExecutionException: An error occurred while executing the command definition. See the inner exception for details. ---> System.Data.SqlClient.SqlException: Cannot roll back addmoney. No transaction or savepoint of that name was found.
Transaction count after EXECUTE indicates a mismatching number of BEGIN and COMMIT statements. Previous count = 1, current count = 2.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.<Reader>b__c(DbCommand t, DbCommandInterceptionContext`1 c)
   at System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext,TResult](TTarget target, Func`3 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   at System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.Reader(DbCommand command, DbCommandInterceptionContext interceptionContext)
   at System.Data.Entity.Internal.InterceptableDbCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   --- End of inner exception stack trace ---
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.Execute(EntityCommand entityCommand, CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.Entity.Core.EntityClient.EntityCommand.ExecuteNonQuery()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunctionCommand(EntityCommand entityCommand)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__4a()
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   at System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass4b.<ExecuteFunction>b__49()
   at System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   at System.Data.Entity.Core.Objects.ObjectContext.ExecuteFunction(String functionName, ObjectParameter[] parameters)
   at DemoSocket.Models.DatabaseGameBaiEntities.truTien(Nullable`1 amount, Nullable`1 id) in D:\C#_project\demo\DemoSocket\DemoSocket\Models\PlayerModel.Context.cs:line 99
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 306


8/15/2019 9:09:26 AM
System.IO.IOException: Unable to write data to the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Send(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamWriter.Flush(Boolean flushStream, Boolean flushEncoder)
   at System.IO.StreamWriter.Write(Char[] buffer, Int32 index, Int32 count)
   at System.IO.TextWriter.WriteLine(String value)
   at DemoSocket.Client.Send(String input) in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 152
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 303


8/15/2019 9:13:45 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/15/2019 9:32:24 AM
System.IO.IOException: Unable to read data from the transport connection: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 208


8/15/2019 9:36:56 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/15/2019 9:37:02 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/15/2019 9:38:46 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/15/2019 9:40:34 AM
System.IO.IOException: Unable to read data from the transport connection: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadLine()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 208


8/15/2019 9:40:35 AM
System.ArgumentException: An item with the same key has already been added.
   at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 297


8/15/2019 9:41:38 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/15/2019 9:41:55 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/15/2019 9:42:22 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/15/2019 9:43:56 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/15/2019 9:52:57 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/15/2019 9:52:59 AM
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 283


8/16/2019 10:28:59 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:38:53 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:39:20 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:40:49 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:41:17 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:41:46 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:44:03 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:45:03 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:46:08 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:48:42 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:51:12 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:53:06 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:55:05 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:58:52 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 11:03:18 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 11:05:40 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 11:10:58 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 11:12:07 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 2:17:37 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 265


8/16/2019 3:23:02 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:24:32 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:26:14 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:27:45 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:29:31 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:33:13 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:38:25 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:40:04 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:42:01 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:43:48 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:49:24 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:51:52 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:52:13 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:54:10 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:56:08 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:56:27 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:57:48 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 3:58:50 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 4:03:59 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 4:13:03 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 4:15:06 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 4:16:47 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 4:18:30 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 4:21:07 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 4:25:36 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 4:27:04 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:25:02 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:25:32 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:26:34 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:31:53 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:33:38 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:38:01 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:39:35 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:41:01 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:42:10 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:45:22 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:45:29 PM
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 283


8/16/2019 8:49:59 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:50:02 PM
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 283


8/16/2019 8:52:55 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:53:44 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:55:33 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:56:37 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 8:58:03 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 9:01:48 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 9:03:39 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 9:04:39 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 9:26:40 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 9:28:40 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 9:31:26 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 9:31:31 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 9:38:23 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 9:38:27 PM
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 283


8/16/2019 10:12:29 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:15:21 PM
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 283


8/16/2019 10:18:10 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:18:13 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:27:07 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 265


8/16/2019 10:27:23 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:45:23 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:45:26 PM
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 283


8/16/2019 10:45:41 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:49:25 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 10:49:28 PM
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 283


8/16/2019 11:01:52 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 265


8/16/2019 11:02:02 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 11:18:11 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/16/2019 11:26:13 PM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/17/2019 12:32:07 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/17/2019 8:51:30 AM
Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: _. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonTextReader.ReadInternal()
   at Newtonsoft.Json.JsonTextReader.Read()
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadForType(JsonReader reader, JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 213


8/17/2019 8:51:37 AM
System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.
   at System.Net.Sockets.Socket.get_RemoteEndPoint()
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 295


8/17/2019 9:08:45 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209


8/17/2019 9:08:55 AM
System.NullReferenceException: Object reference not set to an instance of an object.
   at DemoSocket.Client.DoInBackground() in D:\C#_project\demo\DemoSocket\DemoSocket\Program.cs:line 209

